docs/user/SUMMARY.md

```diff
@@ -14,6 +13,7 @@
     - [Error reporting](user/error-reporting.md)
     - [Using the experiments API](user/experiments-api.md)
 - [Metric types](user/metrics/index.md)
+    - [Metric parameters](user/metrics/metric-parameters.md)
```

Comment:
I'm not sure "Metric parameters" belongs under "Metric types" unless Testing and Adding metrics _also_ belong there.

Reply:
Yeah, I waffled on this a bit.  It also felt wrong in its old location at the top-level.  It's very closely related to the current contents of `user/metrics/index.md`, so maybe should be merged with that?

Reply:
I don't have a strong holistic view of documentation organization beyond that putting it here feels wrong. I'm inclined to leave it where it was for now and file a bug for that sort of shift.

Reply:
Yeah makes sense.  Pinging @brizental for thoughts, since she's done a lot of documentation organization design work.  But if this isn't informed by that, yeah, probably the default position should be to move it back to its original location.

Reply:
I am currently working on a proposal to address this organization. For the current organization though, I agree this should not be in here, but in the "Using the glean SDK" section.